addons/point_of_sale/models/product.py

```diff
@@ -28,9 +28,12 @@ def _unlink_except_open_session(self):
         product_ctx = dict(self.env.context or {}, active_test=False)
         if self.with_context(product_ctx).search_count([('id', 'in', self.ids), ('available_in_pos', '=', True)]):
             if self.env['pos.session'].sudo().search_count([('state', '!=', 'closed')]):
-                raise UserError(_("To delete a product, make sure all point of sale sessions are closed.\n\n"
-                    "Deleting a product available in a session would be like attempting to snatch a"
-                    "hamburger from a customer’s hand mid-bite; chaos will ensue as ketchup and mayo go flying everywhere!"))
+                raise UserError(
+                    _(
+                        "To delete a product, make sure all point of sale sessions are closed.\n\n"
+                        "Deleting a product available in a session would be like attempting to snatch a hamburger from a customer’s hand mid-bite; chaos will ensue as ketchup and mayo go flying everywhere!",
+                    ),
+                )
```

Comment:
Not that it matters much, but I don't think having an extra indentation level just for the gettext helps the readability more than it hurts it, I would have done this:
```suggestion
                raise UserError(_(
                    "To delete a product, make sure all point of sale sessions are closed.\n\n"
                    "Deleting a product available in a session would be like attempting to snatch a hamburger from a customer’s hand mid-bite; chaos will ensue as ketchup and mayo go flying everywhere!",
                ))
```

Reply:
I am updating it with your indentation in the next FW-ports :+1: